### ISSUE TITLE
no tag for latest version

### ISSUE THREAD

**Issue by bosconi** [bp-kelley/descriptastorus#16]
Would it be possible to tag the current head of the master branch and resume tagging each time a PR/groups of PRs is merged down? There are a number of tools that only allow a repo to be pulled based on a tag or release version, which forces forking your repo in order to use them. I would prefer to reference your repo directly.

**Comment by bp-kelley** [comment]
For some reason the builds weren't doing git tag --push, I'll fix this.

**Comment by bosconi** [comment]
Thanks you. Would it be possible to make releases using those tags, going forward? Some tooling can only pull releases, and not just reference tags.

Related question... Are you interested in producing a conda feedstock for descriptastorus? I have made one for now: https://anaconda.org/katanagraph/descriptastorus. There was another one already but it is out of date: https://anaconda.org/RMG/descriptastorus.

I'll continue to update the `katanagraph` one unless you want to make one. If you do, feel free to borrow/adapt the conda-forge recipe that I have staged for it: https://github.com/conda-forge/staged-recipes/compare/main...KatanaGraph:staged-recipes:descriptastorus-feedstock. I was waiting see if you wanted to take this over before submitting it.
